panoptes/routes/api.py

```diff
@@ -66,3 +66,16 @@ def get_job_of_workflow(workflow_id, job_id):
         return jsonify({'msg': 'Workflow not found',
                         'jobs': [],
                         'count': 0})
+
+'''
+/api/delete/<workflow_id>
```

Comment:
This can also go on top of the file with the other API options.

```
/api/workflows
/api/workflow/<workflow_id>
/api/workflow/<workflow_id>/jobs
/api/workflow<workflow_id>/job/<job_id>
/api/delete/<workflow_id> # add this line on top
```